HISTORY.rst

```diff
@@ -1,31 +1,49 @@
-vNEXT
+v0.4.0, 2023-Aug-17
 -----
 
+* The Linux kernel which is supported (>= 5.10.152 5.15.76, 6.0.6, 6.1+)
+* Improved documentation and automated checks
 * Python 3 support.
   Python 2 is no longer supported.
   **NOTE:** If you are running the broker in a Python virtualenv you will have to
   rebuild the virtualenv using a Python 3 interpreter.
   This can be achieved by deleting the old virtualenv folder and recreating it.
   For recreating the virtualenv you can refer to the "Installation" section of
   the broker `installation chapter in the Tunneldigger documentation`_.
+* Added cmake buildsystem to the client.
+* Removed NAT implementation.  
```

Comment:
```suggestion
* Removed NAT-based handling of many client tunnels on the same server port.
  We now rely on the kernel properly distinguishing those UDP sockets.
```

Reply:
Changed it so "we" is not used.  Instead "The broker now relies on the kernel properly distinguishing those UDP sockets." instead.  Done.